students/MiguelErja/index.html

```diff
@@ -1,74 +1,85 @@
-        <meta property="og:image" content="file:///home/miguel/Ironhack/Day1/Personal-WebPage/assets/Personal%20photo.jpeg">
-        <meta property="og:description" content="Personal portfolio. Web development student.">
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:title" content="Miguel Erja González">
+    <meta name="twitter:url" content="hfile:///home/miguel/Ironhack/Day1/Personal-WebPage/index.html">
+    <meta name="twitter:site" content="@ajer1989">
+    <meta name="twitter:description" content="Personal portfolio. Web development student.">
+    <meta name="twitter:image" content="file:///home/miguel/Ironhack/Day1/Personal-WebPage/assets/Personalphoto.jpeg">
 
-        <!-- Twitter -->
-        <meta name="twitter:card" content="summary">
-        <meta name="twitter:title" content="Miguel Erja González">
-        <meta name="twitter:url" content="hfile:///home/miguel/Ironhack/Day1/Personal-WebPage/index.html">
-        <meta name="twitter:site" content="@ajer1989">
-        <meta name="twitter:description" content="Personal portfolio. Web development student.">
-        <meta name="twitter:image" content="file:///home/miguel/Ironhack/Day1/Personal-WebPage/assets/Personalphoto.jpeg">
+  </head>
 
-    </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <a href="file:///home/miguel/Ironhack/Day2/Cohort-Website/cohort0191/index.html" class="home-link link"><i class="fas fa-home"></i>Home</a>
+      </nav>
+      <div class="header-container">
+        <div class="title-container">
+          <h1 class="title">Miguel Erja González</h1>
+          <h2 class="subtitle">Web Developer in Progress</h2>
+        </div>
+        <div class="image-container">
+        </div>
+      </div>
+    </header>
 
-    <body>
-        <header>
+    <main class="main">
+      <section class="main-section">
+        <h2>About me</h2>
+        <p class="description">Lorem ipsum dolor sit amet, id hinc deleniti copiosae sed, tamquam propriae suscipiantur has ex,
+          quo audire consequuntur eu. His dico suscipit ut, te sed quod zril dolores. Ad diceret eleifend
+          corrumpit mei, tale feugiat consequuntur ex cum. Latine fierent deleniti an eam, nec stet liber
+          constituto te, eos ne amet eros malis. Omnesque urbanitas suscipiantur te vel, no blandit convenire
+          accommodare vix. Ad odio docendi scribentur nam, corrumpit gloriatur deseruisse nam ut. Quas lobortis
+          rationibus in cum, augue altera intellegebat ut eum.
+        </p>
+      </section>
 
-            <h1>Miguel Erja González</h1>
-            <h2>Web Developer in Progress</h2>
-            <img src="./assets/Personalphoto.jpeg" alt="personal photograph">
+      <section class="exercises-section">
+        <h2>Exercises</h2>
+        <ul class="exercises-list ulist">
+          <li class="list-item ulist-item">
```

Comment:
you don't have to break the li in this case. You can leave that in the same line but if you want to break it try to be consistent and do that in this way: 
```
<li>
    <a .....></a>
</li>
```